### PR TITLE
Add liteRepository & LiteQueryable extensions

### PR DESCRIPTION
**Pull request by humhei**

### CODE REVIEW

File: LiteDB.FSharp.Tests/Tests.DBRef.fs [hunk 1]
```diff
@@ -7,12 +7,15 @@ open LiteDB
...
+open LiteDB.FSharp.Extensions
 
+let pass() = Expect.isTrue true "passed"
+let fail() = Expect.isTrue false "failed"
 
 let useDatabase (f: LiteRepository -> unit) = 
     let mapper = FSharpBsonMapper()
-    mapper.Entity<Order>().DbRef(convertExpr <@ fun c -> c.Company @> ) |> ignore
-    mapper.Entity<Order>().DbRef(convertExpr <@ fun c -> c.EOrders @> ) |> ignore
+    mapper.DbRef<Order,_>(fun c -> c.Company)
```
Comment (humhei) [inline LiteDB.FSharp.Tests/Tests.DBRef.fs]: Simplify Linq expression

File: LiteDB.FSharp.Tests/Tests.DBRef.fs [hunk 2]
```diff
@@ -22,18 +25,28 @@ let dbRefTests =
...
+        |> LiteRepository.updateItem<Company> { Id = 1; Name = "UpdatedCompanyName" }
+        |> LiteRepository.query<Order> 
+        |> LiteQueryable.``include`` (Expr.prop (fun o -> o.Company))
+        |> LiteQueryable.where (Expr.prop (fun o -> o.Company.Name = "UpdatedCompanyName")) 
+        |> LiteQueryable.toList
+        |> List.length
+        |> function 
+          | 1 -> pass()
+          | _ -> fail()
+
```
Comment (humhei) [inline LiteDB.FSharp.Tests/Tests.DBRef.fs]: Add some f# pipe extension functions

**Comment by humhei** [comment]
Shall we merge it myself ? @Zaid-Ajaj 
Or if you want keep the right to review changes? 😏 
Maybe i am still a new  one to fsharp world and git world

**Comment by Zaid-Ajaj** [comment]
Feel free to merge if you are finished working on the PR :wink:

I did add a review to the code for updateItem, you can comment there or just here if that is easier for you 

When PR is merged, I will publish ASAP

**Comment by humhei** [comment]
I think we should add a script to auto publish to nuget using ci when pr is merged
How do you think about that

**Comment by humhei** [comment]
never mind, as for now only 5 prs existed

**Comment by Zaid-Ajaj** [comment]
> Maybe we should add a script to auto publish to nuget using ci when pr is merged ?

Not every PR is a bug fix or a feature, you might want to just refactor code for better maintainability or even just update README.md. Also I might want to change stuff after a PR so manual publish is better(TM)

**Comment by Zaid-Ajaj** [comment]
Published 1.3.0 :smile: should be available soon on nuget

**Pull request merged**